### PR TITLE
treewide: mark broken packages for 20.03

### LINKED ISSUE #83805: Mark failing to build packages in 20.03 as broken
**Describe the bug**
As Per ZERO Hydra Failure https://github.com/NixOS/nixpkgs/issues/80379, we're supposed to mark every package that fails to build on the stable branch as broken. People can backport fixes to unbreak packages after we cut the final release. This is best done all at once semi-automatically, so this will be a task for @disassembler and I to perform when we are about to cut the final release. (in a PR please :exclamation:)

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
Fixes #83805

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by worldofpeace** [comment]
Stuff I forgot:
```
dvc-with-remotes

libguestfs-with-appliance

libretro stuff???

lincity_ng

linuxPackages

lldb 5 6
https://hydra.nixos.org/build/116359172
https://hydra.nixos.org/build/116361750
```

File: pkgs/tools/package-management/cargo-download/default.nix [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-((import ./Cargo.nix {
+(((import ./Cargo.nix {
   inherit lib buildPlatform buildRustCrate buildRustCrateHelpers fetchgit;
   cratesIO = import ./crates-io.nix { inherit lib buildRustCrate buildRustCrateHelpers; };
 }).cargo_download {}).override {
   crateOverrides = defaultCrateOverrides // {
     cargo-download = attrs: {
       buildInputs = lib.optional stdenv.isDarwin
         darwin.apple_sdk.frameworks.Security;
+      broken = true;
```
Comment (cole-h) [inline pkgs/tools/package-management/cargo-download/default.nix:12]:
```suggestion
```

Comment (worldofpeace) [reply pkgs/tools/package-management/cargo-download/default.nix:12]: we'll just pretend that didn't happen :dancers:

**Comment by worldofpeace** [comment]
yay green

**Comment by jonringer** [comment]
I think there's 2 more:
```
2 package failed to build:
python37Packages.imgaug python38Packages.nixpkgs

1 package built:
python37Packages.nixpkgs
```

**Comment by worldofpeace** [comment]
Need undo marking g2o as broken because https://github.com/NixOS/nixpkgs/pull/85338

**Pull request merged**